src/node/node_state.h

```diff
@@ -822,7 +822,6 @@ namespace ccf
       if (result == kv::ApplyResult::FAIL)
       {
         LOG_FAIL_FMT("Failed to deserialise entry in public ledger");
-        store->rollback(ledger_idx - 1);
```

Comment:
No need for this as the store is about to be rolled back even further, to the previous signature.